docs/container-measurement-design.md

```diff
@@ -164,6 +164,39 @@ Within the measurement list, system processes can be explicitly defined and meas
 
 The concept of container isolation determines whether a container can access the measurements of other containers. In a Kubernetes cluster environment, management pods or containers can also be defined for measurement purposes, ensuring comprehensive coverage of the cluster's integrity.
 
+## Attestation
+
+With the above introduction, the measurement chain for a container can be obtained, encompassing the kernel, services, and workloads. This measurement chain is crucial for ensuring the security and integrity of the containerized environment. The measurement register, which logs these measurements, is recorded in a signed report. By verifying the replay of event logs, we can ensure that the recorded state of the environment is accurate and has not been tampered with. Using these event logs, the environment can be reconstructed and verified, providing a robust mechanism for maintaining trust.
+
```

Comment:
Is it better to add introduction or reference to attestation? Then talk about how to use evidences for attestation.